StreamChatUITestsAppUITests/MockServer/User.swift

```diff
@@ -9,13 +9,15 @@ extension StreamMockServer {
 
     func setUpUser(
         source: [String: Any]?,
-        details: [String: String]
+        details: [String: String] = [:]
     ) -> [String: Any]? {
         var user = source?[JSONKey.user] as? [String: Any]
-        user?[UserPayloadsCodingKeys.id.rawValue] = details[UserPayloadsCodingKeys.id.rawValue]
-        user?[UserPayloadsCodingKeys.name.rawValue] = details[UserPayloadsCodingKeys.name.rawValue]
-        user?[UserPayloadsCodingKeys.imageURL.rawValue] = details[UserPayloadsCodingKeys.imageURL.rawValue]
-        user?["image_url"] = details[UserPayloadsCodingKeys.imageURL.rawValue]
+        if !details.isEmpty {
+            user?[UserPayloadsCodingKeys.id.rawValue] = details[UserPayloadsCodingKeys.id.rawValue]
+            user?[UserPayloadsCodingKeys.name.rawValue] = details[UserPayloadsCodingKeys.name.rawValue]
+            user?[UserPayloadsCodingKeys.imageURL.rawValue] = details[UserPayloadsCodingKeys.imageURL.rawValue]
```

Comment:
Not sure it matters, but the backend works a bit differently. It updates only those fields, that are provided e.g. if the outgoing JSON does not have a value for `name` key, the user's name won't be reset to `nil`

Reply:
Not sure I got you tbh. This has not been done a part of this PR, but happy to discuss it 🙂

Reply:
I'm taking comments as an opportunity to learn more about the mock server. If this missalignment with the real backend does not affect anything - feel free to close